cloudformation/stacks/AutoSpotting/parameters.yaml

```diff
@@ -1,3 +1,8 @@
 ---
 LambdaHandlerFunction: handler.handle
 LambdaS3Bucket: cloudprowess
+LambdaMemorySize: '256'
+ExecutionFrequency: "rate(1 minute)"
+MinOnDemandNumber: '1'
+MinOnDemandPercentage: '50'
```

Comment:
Specifying 2 default values for `MinOnDemandPercentage` & `MinOnDemandNumber` does not seem to make that much sense as Percentage will be (very likely) ignored as `MinOnDemandPercentage` is valid. This is also something we explicated in the documentation, so might be confusing for people reading the code.

Why not defaulting it to 0? as this was the default behaviour before and specify that this can be overridden in the documentation? This way, it wouldn't make a 'breaking-change' for people already using it.